Wiki/NewToFvwm/WhyFvwm/index.md

```diff
@@ -10,27 +10,35 @@ Fvwm is a very old project and at one time was a fairly
 popular window manager, and is now a tiny project still used
 by lots of core users. What is it about fvwm that users like?
 
-+ Fvwm is light weight and fast. Fvwm is built on top of X11,
+{% capture fvwmtxt %}
+<p><font size="3"><b>Fvwm is light weight and fast.</b></font> - Fvwm is built on top of X11,
```

Comment:
First, lets not get in the habit of changing font size this way, it is very inaccessible and bad practice. Go remove all the `<font>` and `</font>` tags. Second never use `<b>`, as this is not accessible. The idea is you are only marking text visually, but this is not proper web design. So go replace all `<b>` and `</b>` with `<strong>` and `</strong>`. This gives proper emphasis, and you shouldn't need to do more than that.

In essence maybe you want what you are making bold and larger to be a header, if that is so, use a header properly.